package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.1",
   "description": "A low overhead circuit breaker for your routes",
   "main": "index.js",
+  "types": "index.d.ts",
   "scripts": {
```

Comment:
Can you also add `"type": "commonjs",`? This will make this package future proof regarding of ESM.

Reply:
sure